### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by VitaliiZhabskyi**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -10,8 +10,15 @@ def count_occurrences(phrase: str, letter: str) -> int:
...
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+
+    return count
```
Comment (vsmutok) [inline app/main.py:24]: try to write it in one line 

use `count()`

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
why did you delete all of the code?

**Comment by VitaliiZhabskyi** [comment]
Hello) can you download it again, I had problems with the Internet and I
deleted it by mistake

чт, 22 черв. 2023 р., 19:10 користувач Vasyl Hnatiuk <
***@***.***> пише:

**Review by Grayder0152 — CHANGES_REQUESTED** [review]
Pls add code

**Pull request closed without merging**